css/css-contain/content-visibility/content-visibility-auto-nested-scroll-ref.html

```diff
@@ -0,0 +1,35 @@
+<!doctype HTML>
+<html class="reftest-wait">
+<meta charset="utf8">
+<title>Content Visibility: content in nested `content-visibility: auto` element can be scrolled to</title>
+<link rel="author" title="Rob Buis" href="mailto:rbuis@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-contain/#content-visibility">
```

Comment:
These usually aren't in the ref file fwiw

Reply:
Good point,. removed.